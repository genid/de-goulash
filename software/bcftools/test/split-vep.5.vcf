##fileformat=VCFv4.2
##contig=<ID=1,length=249250621>
##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|ALLELE_NUM|DISTANCE|STRAND|FLAGS|VARIANT_CLASS|SYMBOL_SOURCE|HGNC_ID|CANONICAL|TSL|APPRIS|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|SOURCE|GENE_PHENO|SIFT|PolyPhen|DOMAINS|miRNA|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|AA_AF|EA_AF|gnomAD_AF|gnomAD_AFR_AF|gnomAD_AMR_AF|gnomAD_ASJ_AF|gnomAD_EAS_AF|gnomAD_FIN_AF|gnomAD_NFE_AF|gnomAD_OTH_AF|gnomAD_SAS_AF|MAX_AF|MAX_AF_POPS|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|LoF|LoF_filter|LoF_flags|LoF_info|CADD_PHRED|CADD_RAW|gnomAD2.1|gnomAD2.1_AF_raw|gnomAD2.1_AF_popmax|gnomAD2.1_AF_afr|gnomAD2.1_AF_amr|gnomAD2.1_AF_asj|gnomAD2.1_AF_eas|gnomAD2.1_AF_fin|gnomAD2.1_AF_nfe|gnomAD2.1_AF_oth|gnomAD2.1_AF_sas">
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	69270	.	A	G	64453.2	PASS	CSQ=G|synonymous_variant|LOW|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.180A>G|ENSP00000334393.3:p.Ser60%3D|180|180|60|S|tcA/tcG|rs201219564|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||||hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&Low_complexity_(Seg):seg&Transmembrane_helices:TMhelix&Prints_domain:PR00237&PROSITE_profiles:PS50262|||||||||||0.8327|0.3603|0.7916|0.8434|0.9983|0.877|0.9112|0.8481|0.9018|0.9983|gnomAD_EAS|||||||||||||||rs201219564|7.81540e-01|9.98354e-01|3.59079e-01|7.93243e-01|8.48238e-01|9.98354e-01|8.82051e-01|9.14571e-01|8.68043e-01|9.00541e-01,G|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000000012|open_chromatin_region||||||||||rs201219564|1||||SNV||||||||||||||||||||||||||0.8327|0.3603|0.7916|0.8434|0.9983|0.877|0.9112|0.8481|0.9018|0.9983|gnomAD_EAS|||||||||||||||rs201219564|7.81540e-01|9.98354e-01|3.59079e-01|7.93243e-01|8.48238e-01|9.98354e-01|8.82051e-01|9.14571e-01|8.68043e-01|9.00541e-01
1	69428	.	T	G	455238	PASS	CSQ=G|missense_variant|MODERATE|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.338T>G|ENSP00000334393.3:p.Phe113Cys|338|338|113|F/C|tTt/tGt|rs140739101|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||deleterious(0.01)|probably_damaging(0.984)|hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&Transmembrane_helices:TMhelix&Prints_domain:PR00237&PROSITE_patterns:PS00237&PROSITE_profiles:PS50262|||0.0190|0.0015|0.036|0.003|0.0497|0.0153|0.003663|0.04571|0.02436|0.003129|0.009178|0.01338|0|0.04895|0.04092|0.02573|0.004991|0.0497|EUR|||||||||||||||rs140739101|2.15376e-02|4.17497e-02|3.34821e-03|9.05886e-03|1.39643e-02|0.00000e+00|4.83761e-02|4.17497e-02|2.93512e-02|4.97165e-03,G|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000000012|open_chromatin_region||||||||||rs140739101|1||||SNV||||||||||||||||||0.0190|0.0015|0.036|0.003|0.0497|0.0153|0.003663|0.04571|0.02436|0.003129|0.009178|0.01338|0|0.04895|0.04092|0.02573|0.004991|0.0497|EUR|||||||||||||||rs140739101|2.15376e-02|4.17497e-02|3.34821e-03|9.05886e-03|1.39643e-02|0.00000e+00|4.83761e-02|4.17497e-02|2.93512e-02|4.97165e-03
1	69438	.	T	C	28054.4	PASS	CSQ=C|synonymous_variant|LOW|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.348T>C|ENSP00000334393.3:p.Tyr116%3D|348|348|116|Y|taT/taC|rs758790416|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||||hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&Prints_domain:PR00237&PROSITE_patterns:PS00237&PROSITE_profiles:PS50262|||||||||||0.0001165|0|0|0|0|0|0.0002685|0|0|0.0002685|gnomAD_NFE|||||||||||||||rs758790416|9.66548e-05|2.64028e-04|0.00000e+00|9.71062e-05|0.00000e+00|0.00000e+00|0.00000e+00|2.64028e-04|0.00000e+00|0.00000e+00,C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000000012|open_chromatin_region||||||||||rs758790416|1||||SNV||||||||||||||||||||||||||0.0001165|0|0|0|0|0|0.0002685|0|0|0.0002685|gnomAD_NFE|||||||||||||||rs758790416|9.66548e-05|2.64028e-04|0.00000e+00|9.71062e-05|0.00000e+00|0.00000e+00|0.00000e+00|2.64028e-04|0.00000e+00|0.00000e+00
1	69453	.	G	A	49.81	PASS	CSQ=A|synonymous_variant|LOW|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.363G>A|ENSP00000334393.3:p.Lys121%3D|363|363|121|K|aaG/aaA|rs2854682|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||||hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&PROSITE_profiles:PS50262|||||||||||7.371e-05|0.0002519|0.0001502|0|0|0|5.667e-05|0|8.675e-05|0.0002519|gnomAD_AFR|||||||||||||||rs2854682|4.73394e-04|3.95946e-04|3.95946e-04|1.45829e-04|0.00000e+00|0.00000e+00|0.00000e+00|6.96728e-05|0.00000e+00|8.71764e-05,A|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000000012|open_chromatin_region||||||||||rs2854682|1||||SNV||||||||||||||||||||||||||7.371e-05|0.0002519|0.0001502|0|0|0|5.667e-05|0|8.675e-05|0.0002519|gnomAD_AFR|||||||||||||||rs2854682|4.73394e-04|3.95946e-04|3.95946e-04|1.45829e-04|0.00000e+00|0.00000e+00|0.00000e+00|6.96728e-05|0.00000e+00|8.71764e-05
1	69469	.	ACAATT	A	305.81	PASS	CSQ=-|frameshift_variant|HIGH|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.380_384del|ENSP00000334393.3:p.Thr127AsnfsTer38|380-384|380-384|127-128|TI/X|aCAATT/a||1||1||deletion|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||||hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&Transmembrane_helices:TMhelix&Prints_domain:PR00245&PROSITE_profiles:PS50262||||||||||||||||||||||||||||||HC||SINGLE_EXON|POSITION:0.418300653594771&PHYLOCSF_TOO_SHORT|||rs1350212330|9.20463e-06|8.71536e-05|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|8.71536e-05
1	69486	.	C	T	3323	PASS	CSQ=T|synonymous_variant|LOW|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.396C>T|ENSP00000334393.3:p.Asn132%3D|396|396|132|N|aaC/aaT|rs548369610|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||||hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&Transmembrane_helices:TMhelix&Prints_domain:PR00245&PROSITE_profiles:PS50262|||0.0002|0|0|0.001|0|0|||2.477e-05|0|0|0|0|0|5.737e-05|0|0|0.001|EAS|||||||||||||||rs548369610|1.90042e-05|5.64223e-05|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|5.64223e-05|0.00000e+00|0.00000e+00
1	69489	.	A	C	1388.94	PASS	CSQ=C|synonymous_variant|LOW|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.399A>C|ENSP00000334393.3:p.Ala133%3D|399|399|133|A|gcA/gcC|rs756368346|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||||hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Superfamily_domains:SSF81321&Transmembrane_helices:TMhelix&Prints_domain:PR00245&PROSITE_profiles:PS50262|||||||||||0.0001428|0|0|0|0|0|0|0|0.000999|0.000999|gnomAD_SAS|||||||||||||||rs756368346|1.15315e-04|1.00384e-03|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|0.00000e+00|1.41475e-05|0.00000e+00|1.00384e-03
1	69496	.	G	A	2206.7	PASS	CSQ=A|missense_variant|MODERATE|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.406G>A|ENSP00000334393.3:p.Gly136Ser|406|406|136|G/S|Ggc/Agc|rs150690004|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||tolerated(0.4)|benign(0.061)|hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Pfam_domain:PF13853&Superfamily_domains:SSF81321&Transmembrane_helices:TMhelix&PROSITE_profiles:PS50262|||0.0010|0.003|0.0014|0|0|0|0.00604|0.0002956|0.0006657|0.006109|0.00143|0|0|0|7.305e-05|0|8.709e-05|0.006109|gnomAD_AFR|||||||||||||||rs150690004|6.64393e-04|6.10650e-03|6.10650e-03|1.53709e-03|0.00000e+00|0.00000e+00|0.00000e+00|7.18061e-05|5.26316e-04|8.74967e-05
1	69511	.	A	G	2.06814e+07	PASS	CSQ=G|missense_variant|MODERATE|OR4F5|ENSG00000186092|Transcript|ENST00000335137|protein_coding|1/1||ENST00000335137.3:c.421A>G|ENSP00000334393.3:p.Thr141Ala|421|421|141|T/A|Aca/Gca|rs2691305|1||1||SNV|HGNC|14825|YES|||CCDS30547.1|ENSP00000334393|Q8NH21||UPI0000041BC1|||tolerated(0.82)|benign(0)|hmmpanther:PTHR26451&hmmpanther:PTHR26451:SF72&Gene3D:1.20.1070.10&Pfam_domain:PF13853&Superfamily_domains:SSF81321&Transmembrane_helices:TMhelix&PROSITE_profiles:PS50262|||||||||0.5441|0.8874|0.9506|0.6074|0.9508|0.9779|0.9995|0.9915|0.9728|0.9499|0.9854|0.9995|gnomAD_EAS|||||||||||||||rs2691305|8.77680e-01|9.99439e-01|6.07496e-01|9.51366e-01|9.76661e-01|9.99439e-01|9.91577e-01|9.72624e-01|9.50621e-01|9.85395e-01
