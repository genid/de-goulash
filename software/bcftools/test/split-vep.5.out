14464	0.0958
